app/src/main/java/org/coepi/android/ui/symptoms/SymptomsViewModel.kt

```diff
@@ -60,8 +73,35 @@ class SymptomsViewModel(private val symptomRepo: SymptomRepo) : ViewModel() {
     fun onChecked(symptom: SymptomViewData) {
         checkedSymptomTrigger.onNext(symptom)
     }
+    fun doSubmitSymptms(){
```

Comment:
Keep names simple. `submitSymptoms` is enough.

Reply:
-        doSubmitSymptms()
+        submitSymptoms()
by the way this method was merged without anyone complainb, but probalby it's hobbit-frodo fault.  repo.doPostSymptoms